crates/libcontainer/src/process/init/process.rs

```diff
@@ -276,6 +277,16 @@ pub fn container_init_process(
         InitProcessError::SyscallOther(err)
     })?;
 
+    // Setup some operations in the network namespace.
```

Comment:
Please make it clearly. What is `some operations`?

Reply:
https://github.com/youki-dev/youki/commit/faf2cdab411d8b8c192c5d21f3d1a6a1539268bb